### PR TITLE
Align IDPF public share encoding with VDAF-13

### PR DESCRIPTION
**Pull request by cjpatton**
Partially addresses #1122.

To simplify the spec, we've coalesced the seeds and payloads into continuous chunks.

### CODE REVIEW

File: src/idpf.rs [hunk 1]
```diff
@@ -690,10 +690,19 @@ where
...
+        //     opaque packed_control_bits[packed_len];
+        //     opaque seed[poplar1.idpf.KEY_SIZE*B];
+        //     Poplar1FieldInner payload_inner[Fi*poplar1.idpf.VALUE_LEN*(B-1)];
+        //     Poplar1FieldLeaf payload_leaf[Fl*poplar1.idpf.VALUE_LEN];
+        // } Poplar1PublicShare;
+        //
+        // Control bits. These need to be written within each byte in LSB-to-MSB order, and
+        // assigned into bytes in big-endian order. Thus, the first four levels will have their
+        // control bits encoded in the last byte, and the last levels will have their control bits
+        // encoded in the first byte.
```
Comment (divergentdave) [inline src/idpf.rs:705]: This comment, and the control bit packing and unpacking, need to be updated as well. We now put the bytes in the opposite order, with the first levels in the first byte.

Comment (cjpatton) [reply src/idpf.rs:705]: I'm confused ... is it just the comment that's out of date, or do you think the bit packing/unpacking code is incorrect? FWIW this is code is currently passing the test vectors on my development branch. (I'm working on a PR for updating the agg param encoding.)

Comment (divergentdave) [reply src/idpf.rs:705]: Ah nevermind, just the comment is wrong, and it's been out of sync with the implementation since #462.

Comment (cjpatton) [reply src/idpf.rs]: Removed the comment!

**Review by divergentdave — APPROVED** [review]

**Pull request merged**